test/test_manifest.json

```diff
@@ -928,6 +929,7 @@
       "md5": "2ade57e954ae7632749cf328daeaa7a8",
       "rounds": 1,
       "link": true,
+      "disabled": true,
```

Comment:
This appears to be the same file: http://web.archive.org/web/20100215194419/http://www.unibuc.ro/uploads_en/29535/10/Cyrillic_Alphabets-Chars.pdf (based on the old link from PR #1787).